snapshot/src/main/java/io/camunda/zeebe/snapshots/impl/FileBasedSnapshotStore.java

```diff
@@ -281,7 +281,6 @@ public Path getPath() {
                 new FileNotFoundException());
           } else {
             try {
-              LOGGER.info("Copying {} to {}", snapshot.getPath(), targetDirectory);
```

Comment:
Lets keep it at least for debugging purposes, so we now which snapshot it tried to recover what do you think?

Reply:
Available snapshot is logged in SnapshotController#recover, isn't that not enough? I think this is just a repeated message without adding any value.

Reply:
Aggreed